modules/ROOT/pages/correlation-id.adoc

```diff
@@ -4,91 +4,35 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: tracing, correlation, id, anypoint, studio
 
-The correlation id is a unique string identification for each event.
-This id is generated when the event is created and starts to be processed by the flow.
-It helps to correlate different log entries to a particular execution.
-The default way the mule runtime generates correlation ids is by generating a java Universally Unique IDentifier (UUID).
+The correlation ID is a unique string identifier for each Mule event. When Mule creates a new event, it also generates a correlation ID before sending the event to the next processor in the flow. This ID enables you to correlate different log entries to a particular execution. By default, Mule generates correlation IDs by generating a Java Universally Unique Identifier (UUID).
 
-As a simple example, we can see the following flow:
+The correlation ID can be used to understand the entire history of an event that resulted in an issue, or an unhandled error, because any error includes the correlation ID of the event that triggered it. Also, the correlation ID is  very helpful while performing a memory analysis of heap dumps.
```

Comment:
```suggestion
The correlation ID can be used to understand the entire history of an event that resulted in an issue, or an unhandled error, because any error includes the correlation ID of the event that triggered it. Also, the correlation ID is very helpful while performing a memory analysis of heap dumps.
```